### PR TITLE
feat: otp service

### PR DESCRIPTION
**Pull request by taufiq**
This PR completes the OTP endpoints (`/otp/verify`, `/otp/generate`) and creates a new Service (`OtpService`) to handle the creation and retrieval of a user's OTP.

### CODE REVIEW

File: app/services/OtpService.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            end_date: {
+              [Op.gte]: new Date(),
+            },
+          },
+        ],
+      },
+      order: [['createdAt', 'DESC']],
+    };
+    try {
+      const result = await db.QuarantineOrders.findOne(params);
```
Comment (christabelneo) [inline app/services/OtpService.js:40]: make this a function in the `QuarantineOrdersService` class and call it within the `generateOtp` function

File: app/services/OtpService.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      if (_.isEmpty(result)) {
+        throw new ApplicationError(`No Quarantine Order with this number, ${contactNumber}, to create OTP for`, 404);
+      }
+      // 2. Generate OTP
+      let otp;
```
Comment (christabelneo) [inline app/services/OtpService.js:45]: do this on the route layer within the `generateOtp` function

File: app/services/OtpService.js (hunk 1 above)
Comment (taufiq) [reply app/services/OtpService.js]: Fixed in 7ae2433

File: app/services/OtpService.js (hunk 2 above)
Comment (taufiq) [reply app/services/OtpService.js]: Fixed in 7ae2433

**Pull request merged**